src/ssh/cockpitsshrelay.c

```diff
@@ -924,12 +919,9 @@ cockpit_ssh_authenticate (CockpitSshData *data)
   int methods_to_try = SSH_AUTH_METHOD_INTERACTIVE |
                        SSH_AUTH_METHOD_GSSAPI_MIC;
 
-#ifdef HAVE_SSH_SET_AGENT_SOCKET
   methods_to_try = methods_to_try | SSH_AUTH_METHOD_PUBLICKEY;
```

Comment:
We should remove the #ifdef but keep flag. Now we rely on the standard ```SSH_AUTH_SOCK``` whereas before we had to setup an fd for the agent.